extensions/podman/src/podman-install.ts

```diff
@@ -550,4 +556,21 @@ class WSL2Check extends BaseCheck {
       return false;
     }
   }
+
+  private async isRebootNeeded(): Promise<boolean> {
+    try {
+      await extensionApi.process.exec('wsl', ['-l'], {
+        env: { WSL_UTF8: '1' },
+      });
+    } catch (error) {
+      console.log(error);
```

Comment:
do we need to log the error like that or we need to log why we have this error and log only if the component is not required ?

Reply:
i just added it for testing and forgot it. It should also be `console.error`. Fixing